pkgs/development/python-modules/django-timezone-field/default.nix

```diff
@@ -13,15 +13,13 @@
 
 buildPythonPackage rec {
   pname = "django-timezone-field";
-  version = "4.2.3";
+  version = "5.0";
   format = "setuptools";
```

Comment:
That is most likely wrong if we require poetry to build the package.

Reply:
Yeah, this used to work, but for 5.0 they dropped the setup.py, so `pyproject` it is.